src/modules/agents/ui/components/columns.tsx

```
@@ -0,0 +1,57 @@
+            <span className="text-sm text-muted-foreground max-w-[200px] truncate capitalize">
+              {row.original.instructions}
+            </span>
+          </div>
+        </div>
+      );
+    },
+  },
+  {
+    accessorKey: "meetingCount",
+    header: "Meetings",
+    cell: ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)